viewer/vueapp/src/components/settings/Settings.vue

```diff
@@ -3238,6 +3311,18 @@ export default {
             friendlyName: 'Dst IP:Dst Port'
           });
 
+          this.integerFields = this.fields.filter(i => i.type === 'integer');
+
+          // attach the full field object to the component's timelineDataFilters from array of dbField2
+          this.timelineDataFilters = [];
+          for (let i = 0, len = this.settings.timelineDataFilters.length; i < len; i++) {
+            let filter = this.settings.timelineDataFilters[i];
+            let fieldOBJ = this.integerFields.find(i => i.dbField2 === filter);
```

Comment:
The UI should always use dbField and not dbField2, only viewer/capture should deal with dbField2